README.template.md

```diff
@@ -364,18 +371,14 @@ ISLANDORA_TAG=x.x.x
 
 Then you can [pull](#pulling) the latest images as described previously.
 
-Then read the **release notes** for the versions between your current version
-and your target version, as manual steps beyond what is listed here will likely
-be required.
-
 Of course **make backups** before deploying to production and test thoroughly.
 
 ## Drupal Development
 
 For local development via the [development profile], an [IDE] is provided which
-can also support the use of [PHPStorm].
+can also support (how?) the use of [PHPStorm].
```

Comment:
What does it mean for a built-in IDE to "support" phpstorm? Where are the instructions to make the built-in IDE work with phpstorm? Or is it the bind-mounted codebase-ish directories which support the use of phpstorm?

Reply:
I "think" it supports the use of phpstrom.